src/MinistryPlatform/Provider.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace SocialiteProviders\MinistryPlatform;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'MINISTRYPLATFORM';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['http://www.thinkministry.com/dataplatform/scopes/all', 'openid', 'offline_access'];
+
+    /**
+     * {@inheritdoc}
```

Comment:
Please add a proper docblock here,`{@inheritdoc}` is only if there is already a parent method